src/components/UniversalNavbarExpanded/MobileNav/MobileNav.js

```diff
@@ -86,6 +86,7 @@ const MobileNav = ({
   LinkComponent,
   singleCta = {},
   animateNavbar,
+  partnerLogo,
```

Comment:
@andriusjurkevicius please add this param to JSDocs above and include specifications for how to use, like image size/format/etc.